<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_841774062.jpg" id="_10_5_622020d_1132928309060_994215_5740" refElementId="_10_5_622020d_1132928309060_994215_5740"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="ServerError" refElementId="_9_5_f720368_1110473546674_245989_8241" refid="_9_5_f720368_1110473546674_245989_8241"/></annotatedElement><body humanName="Body" mode="s,e">A &lt;code&gt;ServerError&lt;/code&gt; is thrown as a result of a remote method
invocation when an &lt;code&gt;Error&lt;/code&gt; is thrown while processing
the invocation on the server, either while unmarshalling the arguments,
executing the remote method itself, or marshalling the return value.

A &lt;code&gt;ServerError&lt;/code&gt; instance contains the original
&lt;code&gt;Error&lt;/code&gt; that occurred as its cause.

@version 1.13, 12/19/03
@author  Ann Wollrath
@since   JDK1.1
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_555864569.jpg" mode="s,e" name="ServerError" refElementId="_9_5_f720368_1110473546674_245989_8241" refid="_9_5_f720368_1110473546674_245989_8241"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="ServerError" refElementId="_9_5_f720368_1110473546674_245989_8241" refid="_9_5_f720368_1110473546674_245989_8241"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>